code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,418 @@
+		)
+		var/organ_or_bodypart_to_spawn = pick(random_bodypart_or_organ)
+		new organ_or_bodypart_to_spawn(src.loc)
+		to_chat(user, span_notice("A piece of flesh crawls out of the painting and flops onto the floor."))
+		// Adds a negative mood event to our heretic
+		user.add_mood_event("heretic_eldritch_hunger", /datum/mood_event/eldritch_painting/desire_heretic)
+
+	// Do they have the mood event added on examine, if so return
+	if ("respite_eldritch_hunger" in user.mob_mood.mood_events)
+		to_chat(user, span_notice("You are still full from your last viewing!"))
+		return
+
+	if (user.has_trauma_type(/datum/brain_trauma/severe/flesh_desire))
+		// Gives them some nutrition
+		user.adjust_nutrition(50)
+		to_chat(user, warning("You feel a searing pain in your stomach!"))
+		user.adjustOrganLoss(ORGAN_SLOT_STOMACH, 5)
+		to_chat(user, span_notice("You feel less hungry, but more empty somehow?"))
+		user.add_mood_event("respite_eldritch_hunger", /datum/mood_event/eldritch_painting/desire_examine)
+
+
+// Specific proximity monitor for The First Desire or /obj/item/wallframe/painting/eldritch/desire
+/datum/proximity_monitor/advanced/eldritch_painting/desire
+	applied_trauma = /datum/brain_trauma/severe/flesh_desire
+	text_to_display = "What an artwork, just looking at it makes me hunger...."
+
+/*
+ * A brain trauma that this eldritch paintings apply
+ * This one is for "The First Desire" or /obj/structure/sign/painting/eldritch/desire
+ */
+/datum/brain_trauma/severe/flesh_desire
+	name = "The Desire for Flesh"
+	desc = "Patient seems to only be able to eat organs or raw flesh for nutrients, also seems to become hungrier at a faster rate"
+	scan_desc = "H_(82882)G3E:__))9R"
+	gain_text = span_warning("I feel a hunger, only organs and flesh will feed it...")
+	lose_text = span_notice("Your stomach no longer craves flesh, and your tongue feels duller.")
+	/// How much faster we loose hunger
+	var/hunger_rate = 15
+
+/datum/brain_trauma/severe/flesh_desire/on_gain()
+	// Allows them to eat faster, mainly for flavor
+	owner.add_traits(TRAIT_VORACIOUS)
+	// We don't want this to be bypassed by Aguesia so if they have it, remove it
+	if(HAS_TRAIT(owner, TRAIT_AGEUSIA))
+		owner.remove_traits(TRAIT_AGEUSIA)
+	// If they have a tongue, make it crave meat
+	var/obj/item/organ/internal/tongue/tongue = owner.get_organ_slot(ORGAN_SLOT_TONGUE)
+	if(tongue)
+		tongue.liked_foodtypes = GORE | MEAT
+	..()
+
+/datum/brain_trauma/severe/flesh_desire/on_life(seconds_per_tick, times_fired)
+	// Causes them to need to eat at 10x the normal rate
+	owner.adjust_nutrition(-hunger_rate * HUNGER_FACTOR)
+	if(prob(2))
+		to_chat(owner, span_notice("You feel a ravenous hunger for flesh..."))
+	owner.overeatduration = max(owner.overeatduration - 200 SECONDS, 0)
+
+/datum/brain_trauma/severe/flesh_desire/on_lose()
+	owner.remove_traits(TRAIT_VORACIOUS)
+	// After loosing this trauma you also loose the ability to taste, sad!
+	owner.add_traits(TRAIT_AGEUSIA)
```

Comment:
Again not how `add_traits`, also a permanent lost to taste *after* getting the trauma cured is.. I guess a little lame, but idk

Reply:
Up to you, but I felt it pretty flavourful like after experiencing these excesses of taste you can no longer go back to how it was